src/routes/index.tsx

```diff
@@ -34,6 +35,15 @@ export default component$(() => {
       <main>
         <div class="flex flex-col gap-10 items-center">
           <h2 class="text-3xl">Split your travel expenses easily</h2>
+          <div>
+            <input
+              type="text"
+              class="w-full shadow border-2 rounded p-1"
+              placeholder="Bill name"
+              value={titleBill.value}
+              onInput$={(_, el) => (titleBill.value = el.value)}
```

Comment:
You can replace it with the shorter syntax, what do you think?

```suggestion
              bind:value={titleBill}
```

Reply:
Sure, it's more clean